go/constants/http.go

```diff
@@ -9,4 +9,5 @@ const (
 	GetRefsPath    = "/getRefs/"
 	HasRefsPath    = "/hasRefs/"
 	WriteValuePath = "/writeValue/"
+  BasePath       = "/"
```

Comment:
Did you forget to run `go fmt`? You should/can set up your editor to do this on save.